cleverhans/utils_th.py

```diff
@@ -62,6 +62,39 @@ def adadelta(loss_or_grads, params, learning_rate=1.0, rho=0.95, epsilon=1e-6):
 
     return updates
 
+def adam(loss_or_grads, params, learning_rate=0.001, beta1=0.9,
```

Comment:
Is this literally just a copy-paste from lasagne? If so, it's probably better to just have this piece of code depend on lasagne